classifier.py

```diff
@@ -13,8 +17,12 @@
 class Classifier(object):
   def __init__(self):
     logging.basicConfig(level=logging.INFO, format='%(asctime)s %(message)s')
-    with open('./data/randomforestclassifier.pickle', 'rb') as fh:
-      self.clf  = load(fh)
+    self.clf  = load(lzma.open('./data/randomforestclassifier.pickle.xz', 'rb'))
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Analysis of this code determined that this line of code contains a resource that might not have closed properly. A resource leak can slow down or crash your system. Programs are strongly recommended to use the built in `with` keyword to open a resource or to use a `try-finally` block to open and close resources explicitly. The [contextlib](https://docs.python.org/3/library/contextlib.html) module provides helpful utilities for using the `with` statement.

[Learn more](https://docs.python.org/3/tutorial/inputoutput.html#reading-and-writing-files)

Reply:
Updated as recommended.

👍🏽